test/e2e/create_test.go

```diff
@@ -246,3 +246,25 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations,
 
 	return err
 }
+
+// waitForMachineDeletion waits for a certain amount of time for the Windows Machines to be deleted
+func (tc *testContext) waitForMachineDeletion() error {
+	machineDeletionTimeLimit := time.Minute * 5
```

Comment:
should `machineDeletionTimeLimit` be a constant?

Reply:
@jrvaldes it could be a constant preferred to keep it is a variable here as we might have to change the value depending on the various platforms in future.